src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -66,6 +66,14 @@ public void toModelType_nullAddress_returnsPerson() throws Exception {
         assertEquals(bensonWithoutAddress, person.toModelType());
     }
 
+    @Test
+    public void toModelType_nullNote_returnsPerson() throws Exception {
+        JsonAdaptedPerson person = new JsonAdaptedPerson(VALID_NAME, VALID_PHONE,
+                VALID_EMAIL, VALID_ADDRESS, VALID_CLIENTSOURCES, null);
+        Person bensonWithoutNote = new PersonBuilder(BENSON).withoutNote().build();
```

Comment:
Just curious, for the withoutNote, couldnt we just not build it without this function>

Reply:
If I didn't understand your question wrongly, the withoutNote() is meant to add in a `null` for BENSON, who originally has a note